gcc/jit/jit-recording.h

```diff
@@ -1500,6 +1503,10 @@ class lvalue : public rvalue
   void set_alignment (unsigned bytes);
   unsigned get_alignment () const { return m_alignment; }
   virtual string * get_name () const { return NULL; }
+  virtual void set_name (const char *new_name) {
+    m_ctxt->add_error (
+      NULL, "cannot change the name of type %s", m_type->get_debug_string ());
```

Comment:
I believe printing the type here would be confusing, no?
On a `vector_access`, the type would be `int` or another integer type. Does that help in any way to know that?

Reply:
I was hoping to get the type name. How would you do it?